app/views/shared/_invitation.html.erb

```diff
@@ -1,9 +1,16 @@
-<form>
-  <% if mobile? %>
-    <%= render partial: "shared/invitations/mobile", locals: local_assigns %>
-  <% else %>
-    <%# if for some reason the user has a really tiny desktop screen fallback to the mobile view" %>
-    <%= render partial: "shared/invitations/mobile", locals: local_assigns %>
-    <%= render partial: "shared/invitations/desktop", locals: local_assigns %>
-  <% end %>
-</form>
+<dt class=<%= mobile? ? "mobile-invitation-content" : "desktop-invitation-content" %>>
+  <%= check_box_tag 'invitations_enabled',true, (@assignment.nil? ? @group_assignment.invitations_enabled : @assignment.invitations_enabled),
```

Comment:
@femmebot do you mean to remove it only from creation or from the assignment settings page as well?
@d12 what are your opinions on this? I think it should remain in the settings page since it is still a setting. Would be weird keeping one specific setting outside of the settings page.

Reply:
I'll defer to @femmebot or the engineering team here to make the correct decision :)